materials/data_structures_R.md

```diff
@@ -5,52 +5,77 @@ title: Data Structures
 language: R
 --- 
 
-## Vectors
+### Vectors
 
-* sequence of values with the same type
-* make one using c(): `sites <- c("a", "a", "b", "b")`
-* explore: `str(sites)`, `length(sites)`
-* Use the Tab key for autocomplete: Let the computer do repetitious work, easier, few mistakes
-* slicing: `sites[1]`, `sites[1:3]`, `1:3` makes a vector so this is the same as
-  `sites[c(1, 2, 3)]` which you can use to get any subset or order you want
-* math:
+* A sequence of values with the same type
+* Assignment: 
+    * `c()`
+
+``` 
+sites <- c("a", "a", "b", "b")
+```
+
+* Useful commands: 
+    * `str(sites)` 
+    * `length(sites)`
+* Slicing: 
+    * `sites[1]` 
+    * `sites[1:3]`
+        * `1:3` makes a vector. So, this is the same as
+    * `sites[c(1, 2, 3)]` 
+        * You can use a vector to get any subset or order you want
+* Math:
+    * `sum()`
+    * `max()`
 
 ```
 counts <- c(0, 2, 2, 6)
 weights <- c(1.5, 2.6, 8.4, 0.2)
-sum(counts)
-max(weights)
 total_weight <- sum(weights)
 ```
 
-* subsetting: `counts[sites == 'a']`
-* `==` means "equal to" in most languages; `=` means assignment
+* Subsetting:
+    * `counts[sites == 'a']`
+        * `==` means "equal to" in most languages. 
+        * Not `=`. `=` is used to assign `function()` arguments.
```

Comment:
`=` *can* be used for assignment more generally, but `<-` is preferred for assignment by the style guide. That nuance is why I chose to specifically state `function()` arguments.